docs/validators/get-started/using-cosmovisor.md

```diff
@@ -36,19 +36,13 @@ git fetch --all
 Check out the tagged release:
 
 ```bash
-git checkout cosmovisor/v1.2.0
-```
-
-Build the cosmovisor binary:
-
-```bash
-make cosmovisor
+git checkout cosmovisor/v1.4.0
 ```
 
-Copy the built binary to your `GOBIN` directory:
+Install the cosmovisor binary:
 
 ```bash
-cp ./cosmovisor/cosmovisor $HOME/go/bin
+go install cosmovisor
```

Comment:
This command doesn't work. Is the goal here to have folks run `go install` without cloning (I provided a full URL of this above)? or to have them clone and build from source (e.g. cloning and then running `make cosmovisor` was in the previous version of this file) ?